pkg/testhelper/testhelper.go

```diff
@@ -33,3 +38,36 @@ func ResultToString(result int) (str string) {
 	}
 	return ""
 }
+
+func SkipIfEmptyAny(skip func(string, ...int), object ...interface{}) {
+	for _, o := range object {
+		s := reflect.ValueOf(o)
+		if s.Kind() != reflect.Slice && s.Kind() != reflect.Map {
+			panic("SkipIfEmpty was given a non slice/map type")
+		}
+
+		if s.Len() == 0 {
+			skip(fmt.Sprintf("Test failed because there are no %s to test, please check under test labels", reflect.TypeOf(o)))
+		}
+	}
+}
+
+func SkipIfEmptyAll(skip func(string, ...int), object ...interface{}) {
+	countLenZero := 0
+	allTypes := ""
+	for _, o := range object {
+		s := reflect.ValueOf(o)
+		if s.Kind() != reflect.Slice && s.Kind() != reflect.Map {
+			panic("SkipIfEmpty was given a non slice/map type")
+		}
+
+		if s.Len() == 0 {
```

Comment:
Minor suggestion: you can get rid of the "countLenZero" variable if you invert the logic returning as soon as a non-zero length object was found:
```
    ...
    allTypes = allTypes + reflect.TypeOf(o).String() + ", "
    if s.Len() != 0 {
        return
    }
  }
  skip(fmt.Sprintf("Test failed because..."))
}
```

Reply:
using countLenZero to go through all the resources since the ones that are zero length are listed in the end